docs/root/intro/version_history.rst

```diff
@@ -28,6 +28,7 @@ Version history
 * config: use Envoy cpuset size to set the default number or worker threads if :option:`--cpuset-threads` is enabled.
 * config: added support for :ref:`initial_fetch_timeout <envoy_api_field_core.ConfigSource.initial_fetch_timeout>`. The timeout is disabled by default.
 * cors: added :ref:`filter_enabled & shadow_enabled RuntimeFractionalPercent flags <cors-runtime>` to filter.
+* ext_authz: removed unnecessary assert that crashes Envoy when GrpcStatus::Unknown is received from a gRPC authorization server.
```

Comment:
I don't think this release note is necessary, but if you want to keep it can you merge master and move to the 1.11.0 section?

Reply:
Sorry, I forgot Envoy is already in 1.10. I will remove this one and fix the other PR :)